dev/secrets/admin-password-secret.yml

```
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: custom-admin-password
+  namespace: awx
+stringData:
+  password: password
```

For other secrets, the values are quoted but not this one
For consistency we should do it here too

```suggestion
  password: 'password'
```